include/glow/Importer/ONNX.h

```diff
@@ -48,28 +48,47 @@ class ONNXModelLoader
   /// Load the network initializers from the GraphProto.
   void loadInitializers(onnx::GraphProto &net);
 
+  /// \returns true if operator \p op can be loaded.
   /// Load the operator \p op into the network. This creates one or more nodes
   /// in the network.
-  void loadOperator(const onnx::NodeProto &op);
+  bool loadOperator(const onnx::NodeProto &op);
 
-  /// Reads a network (weights or structure) from the serialized protocol buffer
-  /// file.
-  bool loadProtoFile(onnx::GraphProto &net, const std::string &filename);
+  /// \returns true if serialized protobuf can be read from the \p filename.
+  /// Loads GraphProto \p net containing network (weights or structure)
+  /// from the serialized protobuf file.
+  bool loadProto(onnx::GraphProto &net, const std::string &filename);
+
+  /// \returns true if serialized protobuf can be constructed from the in-memory
+  /// serialized protobuf.
+  /// Loads GraphProto \p net from the in-memory serialized protobuf \p onnxModel
+  /// with the model size \p onnxModelSize.
+  bool loadProto(onnx::GraphProto &net, void const *onnxModel, size_t onnxModelSize);
+
+  /// \returns true if GraphProto \p net can be loaded from the stream \p iStream.
+  bool loadProto(onnx::GraphProto &net, google::protobuf::io::ZeroCopyInputStream& iStream);
+
+  /// Creates the ONNX model loader with the network which will be stored
+  /// in the \p F. 
```

Comment:
Suggestion:
Creates a ONNX model loader to build \p F.

Otherwise, no *the* before \p F.